internal/handlers/pg/pgdb/database_metadata_test.go

```diff
@@ -62,3 +62,166 @@ func TestDatabaseMetadata(t *testing.T) {
 	})
 	require.NoError(t, err)
 }
+
+func TestRenameCollection(t *testing.T) {
```

Comment:
I'm conflicted here, should we remove these unit tests altogether as there's overlap in the integration tests.

Reply:
I am gonna leave for now and wait on feedback before modifying anything else in this file.